packages/opentelemetry-node/lib/instrumentations.js

```diff
@@ -259,6 +259,15 @@ function getInstrumentations(opts = {}) {
         'OTEL_NODE_DISABLED_INSTRUMENTATIONS'
     );
 
+    // `@opentelemetry/instrumentation-http` defaults to emit old semconv attributes.
+    // Set the default to stable HTTP semconv if not defined by the user (http, http/dup)
+    const semconvOptIn =
+        getStringListFromEnv('OTEL_SEMCONV_STABILITY_OPT_IN') || [];
+    if (!semconvOptIn.some((v) => v.startsWith('http'))) {
+        semconvOptIn.push('http');
+    }
+    process.env.OTEL_SEMCONV_STABILITY_OPT_IN = semconvOptIn.join(',');
```

Comment:
Total nit: but I think checking for the spec-defined `http` and `http/dup` exact strings would be more correct. E.g. if a person fat-fingered: `OTEL_SEMCONV_STABILITY_OPT_IN=httpp` or `=http-dup` or similar.

```suggestion
    if (!semconvOptIn.includes('http') && !semconvOptIn.includes('http/dup')) {
        semconvOptIn.push('http');
        process.env.OTEL_SEMCONV_STABILITY_OPT_IN = semconvOptIn.join(',');
    }
```